docs/source/datasets/dynamicnuclearnet.py

```diff
@@ -0,0 +1,49 @@
+"""
+DynamicNuclearNet
+=================
+
+.. image:: ../../images/tracked.gif
+    :width: 200pt
+    :align: center
+
+DynamicNuclearNet is a training dataset for nuclear segmentation and tracking published in
+Schwartz et al. 2023. The dataset is made up of two subsets, one for tracking and one for
+segmentation.
+
+This dataset is licensed under a `modified Apache license <http://www.github.com/vanvalenlab/deepcell-tf/blob/master/LICENSE>`_ for non-commercial academic use only
+
+The dataset can be accessed using `deepcell.datasets` with a DeepCell API key.
+
+TODO add api key info
+
+Tracking
+^^^^^^^^
+
+Each batch of the dataset contains three components
+
+* X: raw fluorescent nuclear data
+* y: nuclear segmentation masks
+* lineages: lineage records including the cell id, frames present and division links from parent to daughter cells
```

Comment:
@rossbar I can't get these lists to render correctly with rst. Can you take a look? https://deepcell--679.org.readthedocs.build/en/679/data-gallery/dynamicnuclearnet.html#sphx-glr-data-gallery-dynamicnuclearnet-py

Reply:
It looks like this is a sphinx-rtd-theme issue. https://stackoverflow.com/questions/67542699/readthedocs-sphinx-not-rendering-bullet-list-from-rst-file

I was able to resolve it by pinning >=0.5.2 and adding a subsequent pin for m2r2 to avoid a docutils conflict.